.github/workflows/release-tags.yml

```diff
@@ -64,12 +64,17 @@ jobs:
 
       - name: Release under current tag
         run: |
+          PRERELEASE=false
+          if [[ "${RELEASE_VERSION}" =~ "-pre" ]]; then
+            PRERELEASE=true
+          fi
           export "CHANGELOG_FILE=$(mktemp --suffix=.md)"
           echo "CHANGELOG_FILE=${CHANGELOG_FILE}" >> $GITHUB_ENV
           gh api --method POST -H "Accept: application/vnd.github+json" \
             "/repos/${GITHUB_REPOSITORY}/releases/generate-notes" \
             -f tag_name="${RELEASE_VERSION}" \
-            --jq ".body" > "${CHANGELOG_FILE}"
+            --jq ".body" > "${CHANGELOG_FILE}" \
+            -f prerelease="$PRERELEASE"
```

Comment:
This is the wrong place for the flag, the api call only generates the changelog, it's `gh release create` that needs a `--prerelease` flag conditionally added (set PRERELEASE to an empty string by default, to --prerelease if -pre, add it to the commandline **without** quotes to make it work)